config/config.txt

```diff
@@ -584,3 +584,9 @@ TOAST_NOTIFICATION_ON_INIT
 
 ## Causes configuration errors to spit out runtimes
 CONFIG_ERRORS_RUNTIME
+
+## Restricts client uploads to the server, default is 0.5MB
+UPLOAD_LIMIT 524288
+
+## Restricts admin client uploads to the server, default is 5MB
+UPLOAD_LIMIT_ADMIN 5242880
```

Comment:
maybe mention that we define it as 1 KiB being 1024 Bytes, so some people may not think it's on the 1000 Byte system? It's a bit of a weirdity but I know the metric trips people who aren't fully invested in computering so may as well be kind to them

Reply:
> maybe mention that we define it as 1 KiB being 1024 Bytes, so some people may not think it's on the 1000 Byte system? It's a bit of a weirdity but I know the metric trips people who aren't fully invested in computering so may as well be kind to them

is it not defined in bytes? It shows in kilobytes when you try to actually upload file.

Reply:
i think i just worded it a weird way